core/domain/subscription_services.py

```diff
@@ -139,9 +143,9 @@ def unsubscribe_from_creator(user_id, creator_id):
         creator_id: str. The user ID of the creator.
     """
     subscribers_model_creator = user_models.UserSubscribersModel.get(
-        creator_id, strict=False)
+        creator_id)
     subscriptions_model_user = user_models.UserSubscriptionsModel.get(
-        user_id, strict=False)
+        user_id)
```

Comment:
In this function, we are not handling the None case of models(subscribers_model_creator,  subscriptions_model_user). So to remove None from them I removed strict=false.